src/DataTable/Table.js

```diff
@@ -21,6 +21,23 @@ const propTypes = {
     shadow: PropTypes.number
 };
 
+const isString = (s) => typeof(s) === 'string' || s instanceof String;
+
+const getCustomRowProps = (row) => {
```

Comment:
hmm.. Why do we really want to check the received data?
There's a high chance the user know what to do when he uses `mdlRowProps`... I mean.. I hope he knows what he's doing ;p